src/leap/common/testing/basetest.py

```diff
@@ -73,12 +74,13 @@ def setUpEnv(cls):
 
     @classmethod
     def _init_events(cls):
-        cls._server = events_server.ensure_server(
-            emit_addr="tcp://127.0.0.1:0",
-            reg_addr="tcp://127.0.0.1:0")
-        events_client.configure_client(
-            emit_addr="tcp://127.0.0.1:%d" % cls._server.pull_port,
-            reg_addr="tcp://127.0.0.1:%d" % cls._server.pub_port)
+        if flags.EVENTS_ENABLED:
+            cls._server = events_server.ensure_server(
+                emit_addr="tcp://127.0.0.1:0",
+                reg_addr="tcp://127.0.0.1:0")
+            events_client.configure_client(
+                emit_addr="tcp://127.0.0.1:%d" % cls._server.pull_port,
+                reg_addr="tcp://127.0.0.1:%d" % cls._server.pub_port)
```

Comment:
I think I would set the flag `EVENTS_ENABLED` here to `False` making all the tests by defult ignore events. If some specific test needs of the events it can initialize them itself. I don't remember now about any test depending on events, besides the leap.common.event tests but this ones don't use `BaseLeapTest`.